<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LEDControl16FRasPiKDSBelt</class>
 <widget class="QMainWindow" name="LEDControl16FRasPiKDSBelt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LEDControl16FRasPiKDSBelt</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="labelMotorPulseCounter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>XXXX</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>LintTrigger</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>MulInput</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxMulInput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximum">
      <number>16777215</number>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Motor pulse length</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSaveSetting">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>340</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Save setting</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSendFrameTrigger">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Trigger filter</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>111</width>
      <height>141</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Minimum OFF</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxMinOFF">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximum">
      <number>16777215</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Minimum ON</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxMinON">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximum">
      <number>16777215</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>291</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>G-IN</string>
    </property>
    <widget class="QCheckBox" name="checkBoxGIN0">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>IN0</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxGIN3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>IN3</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxGIN2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>IN2</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxGIN1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>IN1</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxGIN4">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>IN4</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxGIN6">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>IN6</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxGIN7">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>110</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>IN7</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxGIN5">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>IN5</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxGIN8">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>IN8</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxGIN15">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>110</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>IN15</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxGIN12">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>IN12</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxGIN10">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>IN10</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxGIN14">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>80</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>IN14</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxGIN11">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>110</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>IN11</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxGIN9">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>50</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>IN9</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxGIN13">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>50</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>IN13</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>10</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>FastIN</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QCheckBox" name="checkBoxFASTIN0">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>IN0</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxFASTIN3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>IN3</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxFASTIN2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>IN2</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxFASTIN1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>IN1</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxFASTIN5">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>IN5</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxFASTIN6">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>IN6</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxFASTIN7">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>110</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>IN7</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxFASTIN4">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>IN4</string>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>350</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Connected</string>
    </property>
   </widget>
   <widget class="QWidget" name="widgetTestMode" native="true">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>190</y>
      <width>491</width>
      <height>151</height>
     </rect>
    </property>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>291</width>
       <height>141</height>
      </rect>
     </property>
     <property name="title">
      <string>G-Out</string>
     </property>
     <widget class="QCheckBox" name="checkBoxoGOUT0">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>61</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>P0</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxoGOUT3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>61</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>P3</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxoGOUT2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>61</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>P2</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxoGOUT1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>61</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>P1</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxoGOUT7">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>110</y>
        <width>61</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>P7</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxoGOUT6">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>61</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>P6</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxoGOUT4">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>61</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>P4</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxoGOUT5">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>61</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>P5</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxoGOUT11">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>110</y>
        <width>61</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>P11</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxoGOUT9">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>50</y>
        <width>61</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>P9</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxoGOUT10">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>80</y>
        <width>61</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>P10</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxoGOUT8">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>61</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>P8</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxoGOUT15">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>110</y>
        <width>61</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>P15</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxoGOUT13">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>50</y>
        <width>61</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>P13</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxoGOUT14">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>80</y>
        <width>61</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>P14</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxoGOUT12">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>20</y>
        <width>61</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>P12</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>0</y>
       <width>161</width>
       <height>141</height>
      </rect>
     </property>
     <property name="title">
      <string>Trigger Out</string>
     </property>
     <widget class="QCheckBox" name="checkBoxoCamFrameTrigger">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>171</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>CamFrameTrigger</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxoCamLineTrigger">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>171</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>CamLineTrigger</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxoCamOut0">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>171</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>CamOut0</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxoCamOut1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>110</y>
        <width>171</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>CamOut1</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QToolButton" name="toolButtonTestMode">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>160</y>
      <width>201</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>TestMode</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
